Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JoiSchemaEnhancer.ts

```diff
@@ -3,6 +3,8 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
+/* eslint-disable no-use-before-define */
+
 // TODO: Need to upgrade to latest joi (https://joi.dev/), which is now top-level and no longer under hapi
```

Comment:
I think you said we would want to _replace_ Joi at some point. Can you create a ticket for that and add the ticket number as a reference in the line above?